server/views/pages/calendarPreview.njk

```diff
@@ -0,0 +1,120 @@
+{% extends "../partials/layout.njk" %}
+
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - Calendar component" %}
+
+{% block content %}
+
+  <h1 class="govuk-heading-l">Choose the visit time</h1>
+  <p>Select one visit time. It will be reserved for 10 minutes.</p>
+  <p>You can book a visit at [establishment name] between [date] and [date].</p>
+  <p>Only times that [prisoner name] can attend are shown.</p>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <nav class="vsip-calendar" aria-label="Visit date selection calendar" role="navigation">
+        {# Loop over months #}
+        {% for monthName, monthData in calendarData.months %}
+          <h2 class="govuk-heading-m">{{ monthName }}</h2>
+
+          {# Day name headings on first month only #}
+          {% if loop.first %}
+            <ul class="govuk-list vsip-calendar__day-headings" aria-hidden="true">
+              <li>M<span>on</span></li>
+              <li>T<span>ue</span></li>
+              <li>W<span>ed</span></li>
+              <li>T<span>hu</span></li>
+              <li>F<span>ri</span></li>
+              <li>S<span>at</span></li>
+              <li>S<span>un</span></li>
```

Comment:
is this done purely for smaller screen purposes? So it can reduce to 'M' instead of trying to fit 'Mon' on the screen?

Reply:
Yes, if you look at line 25 in `vsipCalendar.scss` you'll see that it hides the letters within the `<span>`.